src/pages/search/Search.tsx

```diff
@@ -29,11 +31,65 @@ const Search = () => {
   );
   return (<Fade>
     <Query query={SEARCH_LISTINGS} variables={{ input, ...LISTING_CARD_IMAGE_DIMENSIONS }}>
-      {({ loading, error, data }) =>
-        loading ? <Loading /> :
-        error ? <h2>Error: {error.message}</h2> :
-        <SearchPage listings={data.searchListings} />
-      }
+      {({ loading, error, data }) => {
+        if (loading) {
+          return (
+            <Container tag={Fade} className={VIEWPORT_CENTER_LAYOUT}>
+              <Loading height="8rem" width="8rem" />
+            </Container>
```

Comment:
Oh yay, having the little loading thing stuck in the top-left was bugging me!

Can this be refactored into its own component, e.g. `LoadingPage`? That way we can just drop in an `if (loading) return <LoadingPage />;` wherever we GraphQL

Reply:
sounds good! i'll make a separate PR for this